go.mod

```diff
@@ -43,3 +47,5 @@ require (
 	golang.org/x/sys v0.0.0-20220811171246-fbc7d0a398ab // indirect
 	golang.org/x/text v0.4.0 // indirect
 )
+
+replace github.com/ONSdigital/dp-cache => /home/adegnan/Projects/ons/dp-cache
```

Comment:
Need to remove this

Reply:
Yep.  This requires https://github.com/ONSdigital/dp-cache/pull/5 in order to work, but it'd be nice to get an overview whilst on both parts before merging.